src/main/java/org/sagebionetworks/bridge/services/AdherenceService.java

```diff
@@ -276,4 +276,28 @@ protected void addToMap(Map<String, DateTime> events,
             }
         }
     }
+
+    /* TODO: Add deleteAR method. Should delete AdherenceRecord but might also need
+    *   to update Assessments or Sessions. Also might need to validate input. */
+    public void deleteAdherenceRecords(String appId, AdherenceRecordList recordList) {
+        checkNotNull(recordList);
```

Comment:
This can be for a single record and it's a lot easier if it is. Since it's for testing, they will be manually deleting single records.